.github/workflows/verify-backend-service.yaml

```diff
@@ -0,0 +1,58 @@
+#
+#  Copyright (c) 2024 SAP SE
+#  
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#  
+#  SPDX-License-Identifier: Apache-2.0
+#  
+#  Contributors:
+#        SAP SE - initial API and implementation
+
+---
+name: "Run Tests"
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'mxd/backend-service/**'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+          
+  End-To-End-Tests:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: backendservicedb
+        ports:
+          - 5432:5432
```

Comment:
We run an in memory postgres instance while running tests, right, do we still need it?